src/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// signature_pad's props 
+// (taken from https://github.com/szimek/signature_pad/blob/e2af461bca7ceb7cc5afb8349930908ca19f2f56/src/signature_pad.ts#L23)
+export interface IOptions {
```

Comment:
should this be `IOptions`? or `SignaturePadOptions`? should it be (re-)exported?